SPECS-EXTENDED/squid/squid.spec

```diff
@@ -266,7 +261,7 @@ install -p -D -m 0644 %{SOURCE7} %{buildroot}%{_sysusersdir}/squid.conf
 %{_sysusersdir}/squid.conf
 
 %pre
-%{sysusers_create_compat} %{SOURCE7}
+%{sysusers_create_compat} %{SOURCE9}
 
 for i in %{_var}/log/squid %{_var}/spool/squid ; do
```

Comment:
For tracking purpose, the /var/log/squid might not have been created at this stage, need to set the right ownership in %post stage.